### PR TITLE
mrc-1344 documents index page

### PR DESCRIPTION
**Pull request by hillalex**
Just adds a pretty basic index page for the docs at /docs. We might want to prettify it after feedback from the users.

### CODE REVIEW

File: src/app/src/main/kotlin/org/vaccineimpact/orderlyweb/db/DocumentRepository.kt [hunk 1]
```diff
@@ -5,16 +5,17 @@ import org.vaccineimpact.orderlyweb.db.Tables.ORDERLYWEB_DOCUMENT
...
 
-    override fun getAll(): List<Document> {
+    override fun getAllVisibleDocuments(): List<Document> {
         JooqContext().use {
             db ->
             val rootNodes = db.dsl.selectFrom(ORDERLYWEB_DOCUMENT)
                     .where(ORDERLYWEB_DOCUMENT.PARENT.isNull)
+                    .and(ORDERLYWEB_DOCUMENT.SHOW.eq(1))
                     .fetch()
                     .sortAsc(ORDERLYWEB_DOCUMENT.NAME)
```
Comment (EmmaLRussell) [inline src/app/src/main/kotlin/org/vaccineimpact/orderlyweb/db/DocumentRepository.kt]: Maybe this should order by display name?

File: src/app/src/main/kotlin/org/vaccineimpact/orderlyweb/db/DocumentRepository.kt [hunk 2]
```diff
@@ -27,6 +28,7 @@ class OrderlyDocumentRepository : DocumentRepository {
 
         return db.dsl.selectFrom(ORDERLYWEB_DOCUMENT)
                 .where(ORDERLYWEB_DOCUMENT.PARENT.eq(node.path))
+                .and(ORDERLYWEB_DOCUMENT.SHOW.eq(1))
                 .fetch()
                 .sortAsc(ORDERLYWEB_DOCUMENT.NAME).map {
```
Comment (EmmaLRussell) [inline src/app/src/main/kotlin/org/vaccineimpact/orderlyweb/db/DocumentRepository.kt]: ..this too

File: src/app/templates/documents.ftl [hunk 3]
```diff
@@ -0,0 +1,49 @@
+<@layout>
+    <#macro styles>
+        <link rel="stylesheet" href="${appUrl}/css/style.min.css"/>
+        <style>
+            ul li {
```
Comment (EmmaLRussell) [inline src/app/templates/documents.ftl]: You want to leave this style here, or pull out into the scss?

File: src/app/src/main/kotlin/org/vaccineimpact/orderlyweb/viewmodels/DocumentsViewModel.kt [hunk 4]
```diff
@@ -0,0 +1,17 @@
...
+
+import org.vaccineimpact.orderlyweb.ActionContext
+import org.vaccineimpact.orderlyweb.db.AppConfig
+import org.vaccineimpact.orderlyweb.models.Document
+
+data class DocumentsViewModel(val docs: List<Document>, val appViewModel: AppViewModel) : AppViewModel by appViewModel
+{
+    companion object
+    {
+        val breadcrumb = Breadcrumb("Docs", "${AppConfig()["app.url"]}/docs")
```
Comment (EmmaLRussell) [inline src/app/src/main/kotlin/org/vaccineimpact/orderlyweb/viewmodels/DocumentsViewModel.kt]: Have we got a more offical name than 'Docs'?

File: src/app/templates/documents.ftl (hunk 3 above)
Comment (hillalex) [reply src/app/templates/documents.ftl]: Ok I was being lazy, I'll move it

File: src/app/src/main/kotlin/org/vaccineimpact/orderlyweb/viewmodels/DocumentsViewModel.kt (hunk 4 above)
Comment (hillalex) [reply src/app/src/main/kotlin/org/vaccineimpact/orderlyweb/viewmodels/DocumentsViewModel.kt]: Lol, not sure. I'll ask Kim!

Comment (EmmaLRussell) [reply src/app/src/main/kotlin/org/vaccineimpact/orderlyweb/viewmodels/DocumentsViewModel.kt]: Just need to update this Breadcrumb for the new project-docs route..

**Comment by hillalex** [comment]
> Did you want to put in an actual link to the Docs page in this ticket or another one?

Probably makes sense to wait until there are some actual docs in there to expose the page.

**Pull request merged**